### PR TITLE
read and display data from the database into the leaderboard

### PR DESCRIPTION
**Pull request by PabloJulianRial**
Rady for review

### CODE REVIEW

**Review by Divya-Manivannan1 — CHANGES_REQUESTED** [review]
The code looks good except for a couple of small changes:

1. Please do the database fetching in the leaderboard page. Since we will not be using this info anywhere else, you can leave it in the page rather than bringing it up a level.
--to be done later
3. Also, instead of hardcoding the user profile pictures, use the image URL  from the database. 
--done

File: src/App.tsx [hunk 1]
```diff
@@ -10,23 +10,46 @@ import Leaderboard from "./pages/Leaderboard/Leaderboard";
...
+import { app } from "./firebase";
+import { UserProfile } from "./mockData/mockTribe";
+import { useState, useEffect } from "react";
 
 const App = () => {
+  const [fetchedTribe, setFetchedTribe] = useState<UserProfile[]>([]);
+
+  useEffect(() => {
+    const fetchUsers = async () => {
+      const db = getFirestore(app);
```
Comment (Charlie-robin) [inline src/App.tsx:23]: The db initialisation can be moved to the `firebase.ts` and exported. The reason for this is we do not want to initialise it every time we use it, multiple pages will use the db so it makes sense to create it once.

Have a look at [setup-file-upload](https://github.com/nology-tech/wov-client-project/blob/setup-file-upload/src/firebase.ts) to see who it is don with the storage.

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Great work one minor change please

**Review by Divya-Manivannan1 — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (PabloJulianRial) [reply src/App.tsx]: Hi Charlie, we've moved the db initialization to firebase.ts as requested.

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**